.github/workflows/ttnn-run-sweeps.yaml

```diff
@@ -2,13 +2,41 @@ name: "ttnn - Run sweeps"
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: "0 21 * * *" # This cron schedule runs the workflow at 8:00pm nightly
 
 jobs:
   build-artifact:
     uses: ./.github/workflows/build-artifact.yaml
     secrets: inherit
 
-  ttnn-sweeps:
+  ttnn-generate-sweeps:
+    needs: build-artifact
+    env:
+      TT_METAL_ENV: ${{ vars.TT_METAL_ENV }}
+      ARCH_NAME: ${{ matrix.arch }}
+      ELASTIC_USERNAME: ${{ secrets.SWEEPS_ELASTIC_USERNAME }}
+      ELASTIC_PASSWORD: ${{ secrets.SWEEPS_ELASTIC_PASSWORD }}
+      LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
+    environment: dev
+    timeout-minutes: 30
+    runs-on: build
```

Comment:
add "in-service" label here
or else you could pick up a out of commission builder machine

Reply:
As a side-note, is this suppose to run on our CPU only builders? or does this need a card?
otherwise this will need
for grayskull
runs-on: [E150, in-service, cloud-virtual-machine]
for wormhole:
runs-on: [N150, in-service, cloud-virtual-machine]
runs-on: [N300, in-service, cloud-virtual-machine]

Reply:
Just CPU for this job. Next job in the workflow needs hardware. Can you double check that's correct?

Reply:
It worked when I ran this step before. Just runs-on: ${{matrix.arch}}

Reply:
Can you post a link to the run with the new changes? I don't see any CI job on jdesousa/sweeps-ci-schedule.

if you reference the build/artifact.yaml file. you can see matrix.arch is defined in there, but no arch is defined in this workflow?

For reference, ripped from ./.github/workflows/build-artifact.yaml
```
build-artifact:
    strategy:
      matrix:
        arch: ${{ fromJson(inputs.arch || '["grayskull", "wormhole_b0", "blackhole"]') }}
```